snakeviz/static/snakeviz.js

```diff
@@ -11,15 +11,16 @@ var sv_find_root = function sv_find_root (stats) {
     var diff = callers.subtract(callees);
     if (diff.size !== 0) {
```

Comment:
I wonder if this would have worked if this line was `diff.size === 1` instead? E.g. if the logic should go

```javascript
if (diff.size === 1) {
  return diff.toJS()[0];
} else if (diff.size > 1) {
  return _.max(_.keys(stats), function (s) {
    return stats[s]['stats'][3];
 });
} else {
  throw 'no root found';
}
```